### DISCUSSION TITLE
How do I merge a forked theme into my website?

### DISCUSSION THREAD

**Discussion by Leach456** [community/community#21925] [Other Feature Feedback, Questions, & Ideas]
<p>Hi guys! First off, thank you so much for any help you can offer. I am attempting to implement <a href="https://github.com/artemsheludko/bef" rel="nofollow noopener">this bef theme</a> into my website<a href="http://happyhealthyhydrated.com" rel="nofollow noopener">happyhealthyhydrated.com</a>. </p>
<p>I was able to figure out that I needed to “fork” the theme to utilize it, which I have done.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/113aea7058fe4c5602d8696fc6a342a83ee1000c.png" data-download-href="/uploads/short-url/2sqlihRxN15oduC3j0M5T8RYAwc.png?dl=1" title="Screen Shot 2018-02-06 at 2.36.24 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/36172931/181067557-db5c8def-44be-45f9-91c0-70dcc3616b55.png" alt="Screen Shot 2018-02-06 at 2.36.24 PM.png" data-base62-sha1="2sqlihRxN15oduC3j0M5T8RYAwc" width="690" height="260" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/113aea7058fe4c5602d8696fc6a342a83ee1000c_2_690x260.png, https://aws1.discourse-cdn.com/github/original/1X/113aea7058fe4c5602d8696fc6a342a83ee1000c.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/113aea7058fe4c5602d8696fc6a342a83ee1000c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/113aea7058fe4c5602d8696fc6a342a83ee1000c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-02-06 at 2.36.24 PM.png</span><span class="informations">995×376 40.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>And then within my code, I cloned the fork and uploaded the file with my CNAME file (which is working). But it appears that the theme file isn’t doing anything? </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/79a931e91289fa427724d0ad4f86053fb403c2a0.png" data-download-href="/uploads/short-url/hmggTkQpLC4eo2CRHjALSciGEnu.png?dl=1" title="Screen Shot 2018-02-06 at 2.46.37 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/36172931/181067558-d7870652-c754-4c65-8564-074192d50c73.png" alt="Screen Shot 2018-02-06 at 2.46.37 PM.png" data-base62-sha1="hmggTkQpLC4eo2CRHjALSciGEnu" width="690" height="450" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/79a931e91289fa427724d0ad4f86053fb403c2a0_2_690x450.png, https://aws1.discourse-cdn.com/github/original/1X/79a931e91289fa427724d0ad4f86053fb403c2a0.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/79a931e91289fa427724d0ad4f86053fb403c2a0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/79a931e91289fa427724d0ad4f86053fb403c2a0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-02-06 at 2.46.37 PM.png</span><span class="informations">997×651 53.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Do you have any tips or advice on how to integrate the theme into my website? I am very very confused! Haha. </p>
<p>Thank you!!!</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/leach456">@leach456</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the Pages board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by birdcar** [comment]
<p>Hi <a class="mention" href="/leach456">@leach456</a>!</p>
<p>Ok, so using a theme from another repository can get a bit complex since there are a few “correct” ways to set it up, but I’m confident that we can help you get up and running without too much trouble. Having said that, before I can walk you through how to do this, I’ll need to clarify a couple of things.</p>
<p>First, what is the repository that you would like to publish your Pages site from? Is it this one here:</p>
<p><a href="https://github.com/Leach456/happyhealthyhydrate" rel="nofollow noopener">https://github.com/Leach456/happyhealthyhydrate</a></p>
<p>Second, you’d like to use the Bef theme on your site without modifying it at all, is this correct?</p>
<p>Let me know the answers to those questions and I’ll be able to help you get up and running. :smiley:</p>

**Comment by Leach456** [comment, marked as answer]
<p>Thank you for being willing to help me! I actually was able to figure it out through some YouTube videos and other tutorials. </p>
<p>Thanks :slight_smile:</p>
